Src/controls.h

```diff
@@ -76,6 +76,8 @@ CLI_CONTROL( bool,          DevicePerfCounterTiming,                false, "If s
 CLI_CONTROL( bool,          ITTPerformanceTiming,                   false, "[Note: This control makes ITT calls, but they appear to do nothing!]  If set to a nonzero value, the Intercept Layer for OpenCL Applications will generate ITT-compatible performance timing data.  Similar to DevicePerformanceTiming, this operation may be fairly intrusive and may have side effects; in particular it forces all command queues to be created with PROFILING_ENABLED and may increment the reference count for application events.  ITTPerformanceTiming will also silently create OpenCL command queues that support advanced performance counters if this functionality is available.  This feature will only function if the Intercept Layer for OpenCL Applications is built with ITT support." )
 CLI_CONTROL( bool,          ITTShowOnlyExecutingEvents,             false, "[Note: This control makes ITT calls, but they appear to do nothing!]  By default, when ITTPerformanceTiming is enabled, the Intercept Layer for OpenCL Applications will generate ITT-compatible information for all states of an OpenCL event: when the command was queued, when it was submitted, when it started executing, and when it finished executing.  If ITTShowOnlyExecutingEvents is set to a nonzero value, the Intercept Layer for OpenCL Applications will only generate ITT-compatible instrumentation when an event begins executing and when an event ends executing. Since no information will be displayed about when a command is queued or submitted, this can sometimes make it easier to identify times when the device is idle.  This feature will only function if the Intercept Layer for OpenCL Applications is built with ITT support." )
 CLI_CONTROL( bool,          ChromePerformanceTiming,                false, "If set to a nonzero value, the Intercept Layer for OpenCL Applications will generate device performance timing information in a JSON file that may be used for Chrome Tracing." )
+CLI_CONTROL( bool,			DevicePerfReportMax,					false, "If set to a nonzero value, the Intercept Layer for OpenCL Applications will collect also max values of target platform to .csv with MDAPI counters as a column next to each metric." )
+CLI_CONTROL( bool,			DevicePerfAppendUnits,					false, "If set to a nonzero value, the Intercept Layer for OpenCL Applications will add a row to MDAPI counters with units for each metric." )
```

Comment:
If it makes sense, let's emit the "units" unconditionally, no need to add a control for it.

Reply:
Done.